content/en/docs/best-practices/general-principles/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: General Principles
+description: The general principles for your Istio deployments.
+weight: 1
+icon: best-practices
+---
+
+We've identified the following general principles to help you get the most out
+of your Istio deployments.
+
+## Make your deployment simple
+
+When choosing a [deployment model](/docs/concepts/deployment-models),
+**prefer simplicity over complexity** wherever possible.
+
+## Make your deployment fast
+
+Include clusters in your deployment across the globe for **geographic
+proximity to end-users**. The proximity helps your deployment have low latency.
+
+## Make your deployment reliable
+
+Include clusters in your deployment **across multiple availability regions
+and zones** within each region. This approach helps you avoid global failures.
+
+Follow these recommendations to enhance the reliability of your deployment:
```

Comment:
Do we want to say anything about networks? Deploy the control plane to at least one cluster per network is definitely best practice.

Reply:
Isn't at least one control plane per network a requirement? That is, more than a best practice?

Reply:
@rcaballeromx I don't think this bulleted list belongs in principles, which is the source of this confusion. You're effectively re-stating the actual best practices which will come next.  I think if you just remove this for now you're fine.